src/drivers/camera_trigger/camera_trigger_params.c

```diff
@@ -47,7 +47,7 @@
 *
 * @value 1 GPIO
 * @value 2 Seagull MAP2 (over PWM)
-* @value 3 MAVLink (forward via MAV_CMD_IMAGE_START_CAPTURE)
+* @value 3 MAVLink (Camera Protocol v1)
```

Comment:
FYI, this comment was meaningless - new one reflects the protocol version where is relevant